scripts/vad_to_manifest.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Do we need another script for this? Could this be added as an optional argument in VAD so that VAD can output manifests directly? @fayejf @nithinraok

Reply:
This would enable users to use external vad and combine our speaker embeddings for clustering. And also for testing out oracle vad

Reply:
so has to be an independent script

Reply:
Got it. Thanks!